Makefile

```diff
@@ -99,11 +99,11 @@ delete: ## Delete the controller from your ~/.kube/config cluster
 
 codegen: ## Generate code. Must be run if changes are made to ./pkg/apis/...
 	controller-gen \
-		object:headerFile="hack/boilerplate.go.txt" \
+		object:headerFile="hack/license-boilerplate.txt" \
```

Comment:
Is there a reason we are renaming this? It's pretty standard practice to call this header file `boilerplate.go.txt`